lambdas/service/app.py

```diff
@@ -1208,7 +1212,7 @@ def get_order():
             'token',
             schema.optional({'type': 'string'}),
             description='Reserved. Do not pass explicitly.',
-        ),
```

Comment:
This causes a conflict now.

Reply:
resolved merge conflicts after rebasing with `develop` branch.